.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      bump_type:
+        description: 'Select the version bump type'
+        required: true
+        default: 'patch'
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Loop SRC - ${{ github.sha }}
+        uses: actions/checkout@v4
+        with:
+          path: src
+
+      - name: Read current version
+        id: version
+        working-directory: src
+        run: |
+          version=$(cat VERSION)
+          echo "version=$version" >> "$GITHUB_OUTPUT"
+
+      - name: Create and push tag
+        working-directory: src
+        run: |
+          git config user.name "github-actions"
+          git config user.email "github-actions@users.noreply.github.com"
+          git tag "v${{ steps.version.outputs.version }}"
+          git push origin "v${{ steps.version.outputs.version }}"
+
+      - name: Checkout tag
+        working-directory: src
+        run: |
+          git checkout "v${{ steps.version.outputs.version }}"
+
+      - name: Build release using Docker via Make
+        working-directory: src
+        run: make build_release
+
+      - name: Run tests using Docker via Make
+        working-directory: src
+        run: make test
+
+      - name: Checkout Loop Production branch
+        uses: actions/checkout@v4
+        with:
+          repository: IQSS/ondemand_loop
+          ref: iqss_production
+          path: target
+
+      - name: Copy Loop release build
+        run: |
+          rm -rf target/*
+          cp -R src/application/. target
```

Comment:
why checking out `iqss_production` branch onto `target` if you are going to run `rm -rf target/*` on the next step?

Reply:
My question is why just having one step with this isn't enough:

```yaml
- name: Copy Loop release build
        run: |
          mkdir target
          cp -R src/application/. target
```

there must be something I'm not understanding